cli/resolver.rs

```diff
@@ -327,7 +327,9 @@ impl NpmModuleLoader {
     specifier: &ModuleSpecifier,
     maybe_referrer: Option<&ModuleSpecifier>,
   ) -> Option<Result<ModuleCodeStringSource, AnyError>> {
-    if self.node_resolver.in_npm_package(specifier) {
+    if self.node_resolver.in_npm_package(specifier)
+      || specifier.path().ends_with(".cjs")
```

Comment:
This needs to check if the specifier is a `file:` url, I think? Also, we should error for remote modules that resolve to a cjs media type.